src/plugins/layout/stateprovider.ts

```diff
@@ -129,3 +124,34 @@ export function createGenericLayoutState(node: Node, tag: string): ILayoutState
   };
   return layoutState;
 }
+
+export function shouldIgnoreNode(node: Node, shadowDom: ShadowDom): boolean {
+  let shadowNode = shadowDom.getShadowNode(getNodeIndex(node));
+  let ignore = false;
+  switch (node.nodeType) {
+    case Node.ELEMENT_NODE:
+      let tagName = (node as Element).tagName;
+      if (tagName === "SCRIPT" || tagName === "META") {
```

Comment:
Yeah, we can do that, but i think it deserves a separate sync on what kind of nodes are we willing to ignore and which not, as well as a separate PR.

---
In reply to: [131241375](https://github.com/Microsoft/clarity-js/pull/25#discussion_r131241375) [](ancestors = 131241375)